tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,80 @@
+
+ Feature: CapsulaHub application
+
+ Background:
+    Given a npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a valid port is a number between 1024 and 65535
+    And   two existing config files conf.js and conf1.js located in capsulahub/src/config
+
+ Scenario: Run Capsula-Hub  instance with a valid config file path without specifying the port
+    When  I run the command `capsula-hub run -l <path>` in the relevant directory
```

Comment:
The argument -l will be changed, but it's a minor change - let's wait for the approve of API before changing this in scenario.